### ISSUE TITLE
[Bug] Regarding unnormalize function in the multi-objective tutorial

### ISSUE THREAD

**Issue by yifeng-li** [pytorch/botorch#506]
# 🐛 Bug

https://github.com/pytorch/botorch/blob/master/tutorials/multi_objective_bo.ipynb

Is the use of unnormalize function in functions optimize_qehvi_and_get_observation and optimize_qparego_and_get_observation applicable to other multi-objective problems? Can the authors explain why candidates need to be unnormalized? Is there a theoretical justification I missed? Are candidates normalized in the acquisition function functions?

A related question: in the update_random_observations function, should unnormalize(torch.rand(BATCH_SIZE, 2)) be rand_x = unnormalize(torch.rand(BATCH_SIZE, 2), bounds = problem.bounds) ?

Thanks!

## To reproduce

** Code snippet to reproduce **
```python
new_x =  unnormalize(candidates.detach(), bounds=problem.bounds)
unnormalize(torch.rand(BATCH_SIZE, 2))
```

** Stack trace/error message **
```
// Paste the bad output here!
```

## System information

**Please complete the following information:**
- 
- 
- 
-

**Comment by sdaulton** [comment]
Candidates are generated using `standard_bounds` which are `[0,1]^d` because the model is fit to normalized inputs (transformed to `[0,1]^d`). Therefore, we need to transform the candidates back to the unnormalized search space given by the `problem.bounds`.Yes, `unnormalize(torch.rand(BATCH_SIZE, 2))` should use `bounds=problem.bounds`, but the `update_random_observations` is not actually used in the tutorial--so this isn't really a bug in the tutorial, just broken unused code. I'll remove that. Thanks!

**Comment by yifeng-li** [comment]
Can I replace the standard_bounds below with the bounds of my own problem, say [-1,1]^d ?

    candidates, _ = optimize_acqf(
        acq_function=acq_func,
        bounds=standard_bounds,
        q=BATCH_SIZE,
        num_restarts=20,
        raw_samples=1024,  # used for intialization heuristic
        options={"batch_limit": 5, "maxiter": 200, "nonnegative": True},
        sequential=True,
    )

**Comment by sdaulton** [comment]
So if the search space for your problem is `[-1,1]^d` it would be best to normalize your training inputs (X) to `[0,1]^d` because the standard GP models in botorch have use priors that expect inputs in `[0,1]^d`. The bounds that are passed to `optimize_acqf()` would then be `[0,1]^d`, and after generating candidates, you would unnormalize the candidates from `[0,1]^d` back to `[-1,1]^d. Basically you would replace `problem.bounds` with `[-1,1]^d` everywhere

**Comment by yifeng-li** [comment]
Thanks for the answer! Another question: what is **"nonnegative": True** for? It is used in optimize_qehvi_and_get_observation but not in optimize_qparego_and_get_observation.

**Comment by sdaulton** [comment]
It's is used to determine the strategy for generating initial conditions for the multi-start acquisition optimization: https://github.com/pytorch/botorch/blob/9b0c710657630ec334d82301a061ff43901f54cf/botorch/optim/initializers.py#L71-L76. Botorch uses a different initialization strategy for acquisition functions that are guaranteed to be non-negative. Initially qExpectedHypervolumeImprovement was not included in the list of non-negative acquisition functions in https://github.com/pytorch/botorch/blob/9b0c710657630ec334d82301a061ff43901f54cf/botorch/acquisition/utils.py#L149-L176, so we set the option manually. However, https://github.com/pytorch/botorch/pull/496 added it to the list of non-negative acquisition functions so now setting this option now has no effect. ParEGO uses qExpectedImprovement, which is also in the list of non-negative functions, so the option is not necessary.

**Comment by yifeng-li** [comment]
Got it. Appreciate your answers!
